change/@fluentui-react-accordion-43c26446-45ae-4064-a65f-ceec8761d78e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Explicitely ommit exporting common types from component root",
```

nit: typo
```suggestion
  "comment": "Explicitly omit exporting common types from component root",
```

Same in other change files.